UI/Selenium/appsettings.json

```diff
@@ -49,7 +49,7 @@
       "DefaultElementWait": "20",
       "PipelineElementWait": 30,
       "SaucelabsElementWait": 40,
-      "OneMinuteElementWait": "60",
+      "OneMinuteElementWait": "120",
```

Comment:
if the pipeline is timing out after 20 seconds, I don't think that changing this timeout from 60 to 120 will fix it